src/app/(routes)/faculty/[facultyId]/page.tsx

```diff
@@ -0,0 +1,64 @@
+"use client";
+import { fetchApprovals, updateApproval } from "@/helpers/api";
+import Cookies from "js-cookie";
+import TableComponent from "@/components/TableComponent/TableComponent";
+import generateColumns from "@/components/TableComponent/ColumnMapping";
+import { useState, useEffect } from "react";
+
+const dto = [
```

Comment:
can we change this to
```
status
remarks
salary
     Salary Period
     total CTC
job
     role
     company
         name
     season
         year
         type
         ```
        the backend will now send this.

Reply:
@mittal-ishaan Done changed the dto